Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -760,6 +761,7 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
         quotedMessageId: MessageId? = nil,
         skipPush: Bool = false,
         skipEnrichUrl: Bool = false,
+        restrictedVisibility: [UserId] = [],
```

Comment:
Should we also expose this on `createSystemMessage()` ? I'm not sure, but I think I've heard some customers wanted this for system messages?

Reply:
that's a great point, we should add it yes